libs/rxidb/README.md

```diff
@@ -0,0 +1,220 @@
+const database$: Observable<IDBDatabase> = connectIndexedDb('example_database');
+// Create an Object Store or get an existing one.
+const store$: Observable<IDBObjectStore> = database$.pipe(getObjectStore('example_store'));
+
+store$.subscribe((store) => {
+  // Do something with the store
+});
+```
+
+The `getObjectStore` function takes a string as an argument, which is the name of the Object Store. It returns an Observable that emits an `IDBObjectStore` instance. The second (optional) parameter allows you to configure your Object Store, for example, you can set it to `autoIncrement`.
+
+```typescript
+import { connectIndexedDb, getObjectStore } from '@this-dot/rxidb';
+import { Observable } from 'rxjs';
+
+// Connect to an IndexedDB database (create if the database does not exist)
+const database$: Observable<IDBDatabase> = connectIndexedDb('example_database');
+// Create an Object Store or get an existing one with autoIncrement.
+const store$: Observable<IDBObjectStore> = database$.pipe(
+  getObjectStore('example_store', { autoIncrement: true })
+);
+
+store$.subscribe((store) => {
+  // Do something with the autoIncrement store
+});
+```
+
+### Storing, updating and deleting data in an Object Store
+
+#### setItem()
+
+You can set key-value pairs in an Object Store using the `setItem` operator. This allows you to store or update existing values in the store.
+
+```typescript
+import { connectIndexedDb, getObjectStore, setItem } from '@this-dot/rxidb';
+import { Observable } from 'rxjs';
+
+// Connect to an IndexedDB database (create if the database does not exist)
+const database$: Observable<IDBDatabase> = connectIndexedDb('example_database');
+// Create an Object Store or get an existing one.
+const store$: Observable<IDBObjectStore> = database$.pipe(getObjectStore('example_store'));
+
+function saveToDatabase<T>(key: string, value: T): void {
+  store$.pipe(setItem(key, value)).subscribe();
+}
+
+saveToDatabase('example_key', 'example_value');
+saveToDatabase('example_key2', 1337);
+```
+
+#### addItem()
+
+If you have an Object Store that is set to `autoIncrement`, you can use the `addItem` operator to add a new value to the store.
+
+```typescript
+import { connectIndexedDb, getObjectStore, addItem } from '@this-dot/rxidb';
+import { Observable } from 'rxjs';
+
+// Connect to an IndexedDB database (create if the database does not exist)
+const database$: Observable<IDBDatabase> = connectIndexedDb('example_database');
+// Create an Object Store or get an existing one with autoIncrement.
+const store$: Observable<IDBObjectStore> = database$.pipe(
+  getObjectStore('example_store', { autoIncrement: true })
+);
+
+function saveToDatabase<T>(key: string, value: T): void {
+  store$.pipe(addItem(key, value)).subscribe();
+}
+
+saveToDatabase('example_value');
+saveToDatabase(1337);
+```
+
+#### deleteItem()
+
+You can use the `deleteItem()` operator to delete a value from the store.
```

Comment:
We should specify in the docs what happens when `deleteItem` is called and the key doesn't exisits.